### PR TITLE
Add tctl users reset to cli docs

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/5.0/cli-docs.md [hunk 1]
```diff
@@ -579,6 +579,30 @@ tctl users rm sally,tim
 # Removes users sally and tim
 ```
 
+## tctl users reset
+
+Reset local user account password and two factor with expiring link to populate vaues. **Usage**: `tctl users reset <account>`
```
Comment (webvictim) [inline docs/5.0/cli-docs.md:584]:
```suggestion
Reset local user account password and any associated second factor with expiring link to populate values. **Usage**: `tctl users reset <account>`
```

File: docs/5.0/cli-docs.md [hunk 2]
```diff
@@ -579,6 +579,30 @@ tctl users rm sally,tim
...
+| Name | Default Value(s) | Allowed Value(s) | Description
+|------|---------|----------------|----------------------------|
+| `--ttl` | 8h | relative duration like 5s, 2m, or 3h | Set expiration time for token, default is 8h0m0s, maximum is 24h0m0s
+
+
+### Examples
+
+``` bsh
+# tctl users reset jeff 
+User stevenexample has been reset. Share this URL with the user to complete password reset, link is valid for 8h0m0s:
```
Comment (webvictim) [inline docs/5.0/cli-docs.md:601]:
```suggestion
User jeff has been reset. Share this URL with the user to complete password reset, link is valid for 8h0m0s:
```

File: docs/5.0/cli-docs.md [hunk 3]
```diff
@@ -579,6 +579,30 @@ tctl users rm sally,tim
...
+https://teleport.example.com:3080/web/reset/8a4a40bec3a31a28db44fa64c0c70ca3
+# Resets jeff password, two factor token.  Jeff populates the password and confirms token with link.
```
Comment (webvictim) [inline docs/5.0/cli-docs.md:603]:
```suggestion
# Resets jeff's password and any associated second factor.  Jeff populates the password and confirms token with link. 
```

**Comment by webvictim** [comment]
Also, turn on something in your editor that removes trailing whitespace!

(or run `make docs-fix-whitespace`)

**Comment by stevenGravy** [comment]
> Also, turn on something in your editor that removes trailing whitespace!
> 
> (or run `make docs-fix-whitespace`)

Fixed, will use next time. @webvictim

**Review by webvictim — APPROVED** [review]

**Pull request merged**